### PR TITLE
Mention argless `#annotations` method in annotation docs

### PR DESCRIPTION
**Pull request by Blacksmoke16**
Long overdue mention of https://github.com/crystal-lang/crystal/pull/9326.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 site/
 .venv/
+__pycache__/
```
Comment (straight-shoota) [inline .gitignore:3]: **question:** This seems to be an unrelated change. What's the motivation or where does it come from?

Comment (Blacksmoke16) [reply .gitignore:3]: It get created automatically when I was running `make serve`, but with `mkdocs` coming from `.venv`.

Comment (straight-shoota) [reply .gitignore:3]: I guess it's fine to add this to `.gitignore`. But let's drop it here and do a separate PR for that.
/cc @oprypin

**Review by straight-shoota — APPROVED** [review]

**Comment by Blacksmoke16** [comment]
Bump, this should be good to go as well.

**Comment by straight-shoota** [comment]
Sorry, I forgot to add this to the project and thus lost track of it (unfortunately, GH only allows automatic adding from one repo 🤷). Feel free to add it yourself next time. Or merge directly if you feel lucky 😆
